src/test/java/LoggerExtension.java

```diff
@@ -0,0 +1,42 @@
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.extension.AfterEachCallback;
+import org.junit.jupiter.api.extension.AfterTestExecutionCallback;
+import org.junit.jupiter.api.extension.BeforeAllCallback;
+import org.junit.jupiter.api.extension.BeforeEachCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.api.extension.TestInstancePostProcessor;
+
+import java.util.logging.Logger;
+
+public class LoggerExtension implements TestInstancePostProcessor, BeforeAllCallback, BeforeEachCallback, AfterTestExecutionCallback, AfterEachCallback {
+    @Override
+    public void afterEach(ExtensionContext context) throws Exception {
+        Logger LOGGER = Logger.getLogger(JunitPracticalTask1.class.getName());
```

Comment:
Firsly, we use UPPER_CASE only for constants or static fields.
Secondly, your extension might work with any class not only `JunitPracticalTest1`, so it's better to extract test class from `ExtensionContext`